plugins/templates/labhub/hello-world.jinja2.md

```diff
@@ -0,0 +1,11 @@
+Welcome @{{ target }}! :tada:
+
+A developer or maintainer will soon invite you to the orgazinzation.
+
+Meanwhile, to get started, please follow our [newcomers guide] (https://coala.io/newcomer) and skip instructions using corobo. Most issues will be explained there and in linked pages - it will save you a lot of time, just read it. *Really.*
+
+After you've chosen you issue, ask a developer or maintainer to assign that issue to you.
```

Comment:
`After you've chosen you issue`?

Reply:
`developer or maintainer` --> `maintainer`

Reply:
I think I should remove this line, since we are giving newcomers right to assign themselves issues after they've been invited.